general/app/development/links/links.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Links in the Moodle App
+sidebar_label: Links
+sidebar_position: 1
+tags:
+ - Moodle App
+---
+
+## Pressing a link inside the app
+
+When a user presses a link in the Moodle app, the behaviour changes depending on whether the URL is supported by the app or not:
+
+- If the URL belongs to the same site and it's supported by the app, then the app will try to open the corresponding page inside the app. For example if the user presses a link to an assignment then the assignment will be opened inside the app if possible.
+- If it's a URL pointing to an external site or it's a URL not supported by the app then it will be opened in a browser. For example admin settings aren't supported by the app, so clicking a link to an admin settings page will open the page in browser.
+
+## Extending the list of supported URLs
+
+The app has a defined list of supported URLs. If you have a plugin adapted to work in the app and you want to support links to your plugin you will need to create a Link Handler. For more information and examples about this, please see the [Link handlers](../plugins-development-guide/index.md#link-handlers) documentation.
+
+## Opening links in an embedded browser
+
+To open a link in an embedded browser instead of an external browser you can use the `data-open-in` attribute:
+
+```html
+<a href="https://domain.com" data-open-in="app">
```

Comment:
What are the possible values of `data-open-in`? `app` and `external`? I guess we should document it as well, even if `external` is already the default value.

Reply:
Right now the only possible value is `app`, all other values open the browser.